src/mongo/client/dbclient.cpp

```diff
@@ -1416,6 +1424,10 @@ bool DBClientWithCommands::eval(const string& dbname,
 }
 
 bool DBClientWithCommands::eval(const string& dbname, const string& jscode) {
```

Comment:
I should clarify that I'm not saying you need to change it. I don't think we care about an extra symbol.